src/music.c

```diff
@@ -339,7 +341,14 @@ void SDLCALL music_mixer(void *udata, Uint8 *stream, int len)
         }
 
         if (music_playing->interface->GetAudio) {
-            int left = music_playing->interface->GetAudio(music_playing->context, stream, len);
+            int left;
+            if (master_vol >= MIX_MAX_VOLUME || !mix_music_buffer) {
```

Comment:
Can `master_volume` ever be greater than `MIX_MAX_VOLUME`?

Reply:
Not intended to be, but just in a case.